### PR TITLE
validation substitution

### PR DESCRIPTION
**Pull request by btry**
document the validation substitution feature

https://github.com/glpi-project/glpi/pull/12504

### CODE REVIEW

File: source/modules/user-settings/authorized-substitutes.rst
Comment (cconard96) [inline source/modules/user-settings/authorized-substitutes.rst:8]:
```suggestion
The user may also be a delegatee of another user. In this case, they will see a table of their delegators and the date range of each delegation.
```

**Review by cconard96 — CHANGES_REQUESTED** [review]

File: source/modules/user-settings/authorized-substitutes.rst
Comment (btry) [reply source/modules/user-settings/authorized-substitutes.rst:6]:
```suggestion
The user can choose one or more accounts to delegate the validation task to. He may optionnaly limit the date range of the delegation by setting one or both date limits.
```

I think we cannot replace "he" with "they" because it refers to the user at the beginning of the 1st sentence.

File: source/modules/user-settings/authorized-substitutes.rst
Comment (btry) [reply source/modules/user-settings/authorized-substitutes.rst:8]:
```suggestion
The user may also be a delegatee of another user. In this case, he will see  a table of his delegators and the date range of each delegation." 
```

Again, i don't think that "he should be replaced with "they" for the same reason. If there is an ambiguity, then i should find an other way to describe the features.

File: source/modules/user-settings/authorized-substitutes.rst
Comment (cconard96) [reply source/modules/user-settings/authorized-substitutes.rst:6]: "They" can refer to a multiple people or it can also be used to refer to a single person.

Without getting too politically correct, in English we tend to default to gender neutral language when it is not known as opposed to French which seems like it defaults to masculine forms of words.

Comment (btry) [reply source/modules/user-settings/authorized-substitutes.rst:6]: OK, Understood.

**Review by cconard96 — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]
Should we wait for 10.1 to be released before merging this PR?

**Comment by btry** [comment]
The feature has been merged in 10.1, then yes we should publish for this version only

**Comment by trasher** [comment]
There is only one doc for all releases right now. So we should not merge this one for the moment.

By the way, please add sphinx version added on relevant parts

**Comment by JohnnyMacao** [comment]
hi, is this feature usable in glpi 10.0.7? thank you.

**Comment by cedric-anne** [comment]
@JohnnyMacao 

It will be available in GLPI 10.1. No release date is planned yet.
image

**Pull request merged**
